### PR TITLE
dojson: use logger, not warnings

### PR DESCRIPTION
**Pull request by jacquerie**
Sentry: https://sentry.cern.ch/inspire-sentry/inspire-labs-qa/group/601658/

### CODE REVIEW

File: inspirehep/dojson/hep/fields/bd5xx.py [hunk 1]
```diff
@@ -150,11 +155,14 @@ def thesis2marc(self, key, value):
 def abstracts(self, key, value):
     """Summary, Etc.."""
     if isinstance(value.get('a'), (list, tuple)):
-        import warnings
```
Comment (david-caro) [inline inspirehep/dojson/hep/fields/bd5xx.py]: The internal import is not needed anymore?

Comment (david-caro) [inline inspirehep/dojson/hep/fields/bd5xx.py]: Otherwise LGTM

**Comment by kaplun** [comment]
Ahi, ahi! Travis disagrees. How come you haven't run locally the unit tests before pushing? :stuck_out_tongue_closed_eyes:

File: inspirehep/dojson/hep/fields/bd5xx.py (hunk 1 above)
Comment (jacquerie) [inline inspirehep/dojson/hep/fields/bd5xx.py]: Yup, the `logging` module is used instead.

**Comment by jacquerie** [comment]
> Ahi, ahi! Travis disagrees. How come you haven't run locally the unit tests before pushing? 😝

I deserve this. Don't do like me, kids!

**Pull request merged**